cds_ils/ldap/tasks.py

```diff
@@ -28,5 +29,6 @@ def synchronize_users_task():
         result = update_users()
         log.set_succeeded(*result)
     except Exception as e:
+        db.session.rollback()
```

Comment:
👍 great! Were you able to validate that it actually fixes the issue?

Reply:
I tested it this morning and it was in fact the cause of the issue of the tasks being left "RUNNING". We had a test that covered failures, but database errors are different in the sense that they have side-effects (and we must rollback the changes when one occurs, before committing further changes). The errors are now correctly reported in the database.